### PR TITLE
Validate cmd integration test

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
Added integration test to `validate` command

### CODE REVIEW

File: demisto_sdk/tests/integration_tests/validate_integration_test.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+def test_integration_validate():
+    results = run(
+        [PYTHON_CMD, MAIN_MODULE_PATH, VALIDATE_CMD, "-p", PACK_INTEGRATION_PATH],
+        stderr=PIPE,
+        stdout=PIPE,
+        encoding='utf-8',
+        cwd=join(DEMISTO_SDK_PATH, "tests", "test_files", "content_repo_example")
+    )
+    stdout = results.stdout
```
Comment (yaakovi) [inline demisto_sdk/tests/integration_tests/validate_integration_test.py:21]: Also check status code is 1 if you expect a failure

Comment (Itay4) [reply demisto_sdk/tests/integration_tests/validate_integration_test.py:21]: done

**Comment by ronykoz** [comment]
@Itay4 i think that we should add an happy flow case as well

**Comment by yaakovi** [comment]
Why coverage decreased?

**Comment by Itay4** [comment]
> @Itay4 i think that we should add an happy flow case as well

@ronykoz added positive test for incident field

File: demisto_sdk/tests/integration_tests/validate_integration_test.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+from demisto_sdk.commands.common.git_tools import git_path
+
+DEMISTO_SDK_PATH = join(git_path(), "demisto_sdk")
+MAIN_MODULE_PATH = join(DEMISTO_SDK_PATH, "__main__.py")
+PYTHON_CMD = "python"
+VALIDATE_CMD = "validate"
+
+
+def test_integration_validate_integration_negative():
```
Comment (anara123) [inline demisto_sdk/tests/integration_tests/validate_integration_test.py:12]: add description to the test.
```
Given
- integration FeedAzure with old docker image 

When
- running validation on it

Then
- ensure validation is failed
- the validation error is on old docker image tag
```

Comment (Itay4) [reply demisto_sdk/tests/integration_tests/validate_integration_test.py:12]: done

**Pull request merged**